src/encoded/schemas/changelogs/file.md

```diff
@@ -17,7 +21,7 @@
 
 ### Minor changes since schema version 12
 
-* New enumerations that could be posted only by the DCC were added in output_types: ""redacted alignments" and "redacted unfiltered alignments"; both enumerations are categorized as "alignments".
+* New enumerations that could be posted only by the DCC were added in output_types: "redacted alignments" and "redacted unfiltered alignments"; both enumerations are categorized as "alignments".
```

Comment:
Sorry, fixes unrelated to the ticket are discouraged. Please remove, and make a new branch including a description of the desired change.

Reply:
Removed the change